### PR TITLE
networking: Update Multus example to use bridge CNI

### PR DESCRIPTION
**Pull request by ormergi**
**What this PR does / why we need it**:
Update the 'Interface And Networking' docs, Multus example to use bridge CNI.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

File: docs/virtual_machines/interfaces_and_networks.md [hunk 1]
```diff
@@ -1100,8 +1100,9 @@ spec:
   config: '{
             "cniVersion": "0.3.1",
             "name": "br-spoof-check",
-            "type": "cnv-bridge",
```
Comment (AlonaKaplan) [inline docs/virtual_machines/interfaces_and_networks.md:1102]: Please replace cnv-bridge with bridge all over the file.

Comment (ormergi) [reply docs/virtual_machines/interfaces_and_networks.md:1102]: Done

File: docs/virtual_machines/interfaces_and_networks.md [hunk 2]
```diff
@@ -1079,17 +1080,13 @@ There are two known CNI plugins that support mac-spoof-check:
...
 - [sriov-cni](https://github.com/openshift/sriov-cni):
   Through the `spoofchk` parameter .
-- cnv-bridge: Through the `macspoofchk`.
-
-> **Note:** `cnv-bridge` is provided by
-  [CNAO](https://github.com/kubevirt/cluster-network-addons-operator).
-  The [bridge-cni](https://github.com/containernetworking/plugins) is planned
-  to support the `macspoofchk` options as well.
+- [bridge-cni](https://github.com/containernetworking/plugins):
+  Through the `macspoofchk`.
```
Comment (EdDev) [inline docs/virtual_machines/interfaces_and_networks.md:1084]: What is this?

**Review by EdDev — CHANGES_REQUESTED** [review]

File: docs/virtual_machines/interfaces_and_networks.md [hunk 3]
```diff
@@ -1079,17 +1080,13 @@ There are two known CNI plugins that support mac-spoof-check:
 
 - [sriov-cni](https://github.com/openshift/sriov-cni):
   Through the `spoofchk` parameter .
-- cnv-bridge: Through the `macspoofchk`.
-
-> **Note:** `cnv-bridge` is provided by
-  [CNAO](https://github.com/kubevirt/cluster-network-addons-operator).
-  The [bridge-cni](https://github.com/containernetworking/plugins) is planned
-  to support the `macspoofchk` options as well.
+- [bridge-cni](https://github.com/containernetworking/plugins):
```
[the remark this answers is not in the document]
Comment (ormergi) [reply docs/virtual_machines/interfaces_and_networks.md]: Thanks, fixed.

File: docs/virtual_machines/interfaces_and_networks.md (hunk 2 above)
Comment (EdDev) [reply docs/virtual_machines/interfaces_and_networks.md:1084]: But this line I commented on is left here.

Comment (EdDev) [reply docs/virtual_machines/interfaces_and_networks.md:1084]: OK, now I understand. I think you just need to sync it with the previous text. i.e. `Through the macspoofchk parameter`.

Comment (ormergi) [reply docs/virtual_machines/interfaces_and_networks.md]: Thanks for the heads up, fixed.

**Review by EdDev — APPROVED** [review]
Thanks!

**Pull request merged**